Integrations/ESPHome/AIR-1.yaml

```
@@ -346,6 +348,12 @@ text_sensor:
         return std::string("Very abnormal");} 
       else {
         return std::string("Extremely abnormal");}
+  - platform: template
+    name: "Firmware Version"
+    icon: mdi:pound
```

Looks great and tested on my AIR-1. Thoughts on adding this to the config section instead of normal sensors? I'll apply this to our other products once we've got this one.

```suggestion
    icon: mdi:pound
    entity_category: "config"
```

---

Great idea!